test/cli/test-other.py

```diff
@@ -825,3 +825,25 @@ def test_file_duplicate_2(tmpdir):
         '3/3 files checked 0% done'
     ]
     assert stderr == ''
+
+
+def test_cgf_with_relative_path(tmpdir):
+
+    test_file = os.path.join(tmpdir, 'test.c')
+    with open(test_file, 'wt'):
+        pass
+
+    test_cfg = os.path.join(tmpdir, '../../cppcheck.cfg')
+    with open(test_cfg, 'wt') as f:
+        f.write("""
+                {
+                    "addons": [],
+                    "productName": "Cppcheck Premium 1.2.3.4",
+                    "about": "Cppcheck Premium 1.2.3.4"
+                }
+                """)
+
+    args = [test_file]
+
+    _, _, stderr = cppcheck(args)
+    assert stderr == ''
```

Comment:
This is actually not testing anything. It needs to check the output for something that would actually indicate the config was loaded.

Reply:
it will now check that the proper product_name is reported when --version is used.